cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/orion/OrionBackend.java

```diff
@@ -48,4 +48,28 @@
     JsonResponse subscribeContext(String cygnusSubscription, boolean xAuthToken, 
             String token) throws Exception;
     
+    /** 
+     * Unsubscribe from Orion with a given subscription id (Orion v1)
+     * 
+     * @param subscriptionId 
+     * @param xAuthToken 
+     * @param token
+     * @throws Exception
+     * @return response
+     */
+    public JsonResponse deleteSubscriptionV1(String subscriptionId, boolean xAuthToken, 
```

Comment:
The boolean parameter may be avoided: if token is null then it means no token must be used.